caravel/assets/javascripts/dashboard/components/Modal.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+
+const propTypes = {
+  modalId: PropTypes.string.isRequired,
+  title: PropTypes.string,
+  modalContent: PropTypes.node,
+  customButtons: PropTypes.node
+};
+
+class Modal extends React.Component {
```

Comment:
it would be great to move this component to `caravel/assets/javascripts/components/Modal.jsx` so we can reuse it across the app.